gate-core/src/main/groovy/com/netflix/spinnaker/gate/filters/FiatSessionFilter.groovy

```diff
@@ -76,6 +76,8 @@ class FiatSessionFilter implements Filter {
             SecurityContextHolder.clearContext()
           }
         }
+      } else {
+        log.debug("Authenticated user was not present in authenticated request. Check authentication settings.")
```

Comment:
Maybe this should be a warn log?

Reply:
Yeah, I was on the fence on this, but since you said it too I agree. I've pushed an update to change it to warn. After all, I can't think of a valid scenario where an AuthenticatedRequest wouldn't have a user associated with it. Good call.